### ISSUE TITLE
how to query dotnet version for func app

### ISSUE THREAD

**Issue by boris-tarasov-maersk** [Azure/azure-functions-core-tools#2735]
I had a case when some external libs are not working correctly, and I had an ask to provide output of 'dotnet --version'
I tried to do from ssh consol in Azure Fun app advance tool and by running my func in a docker image and getting a terminal access.
it seems that dotnet executable is not included as self standing binary that can be called with func app. How to run 'dotnet --version' in such situation?

the reason I also tried docker image is that issue I have is related to running func app in linux env and my machine is runnig Win OS>

**Comment by apawast** [comment]
@boris-tarasov-maersk can you please share the OS and plan you are using?

**Comment by boris-tarasov-maersk** [comment]
@apawast 
thank you for responding to my request. 
I had issue running azure functions with some native libraries, case I have described here and the one you recently viewed:
https://github.com/Azure/azure-functions-docker/issues/520

my goal with this task was to be able to query dontet --version command.
using standart azure function docker image from Microsoft it seems dotnet command was not present to call directly from a terminal. I spend good time trying to fine dontnet executable. The one that is present on standart docker image for dotnet sdk.
so I realized it would be nice to be able to qeury such information programatically or in any other way.

I could find such solution so I asked.

when I run uname -a on in that docker container I get following output:
`root@2c8dfdacf9fb:/home/site/wwwroot# uname -a
Linux 2c8dfdacf9fb 5.10.16.3-microsoft-standard-WSL2 #1 SMP Fri Apr 2 22:23:49 UTC 2021 x86_64 GNU/Linux`
